PWGHF/vertexingHF/vHFML/AliAnalysisTaskSEDs.cxx

```diff
@@ -1670,7 +1701,7 @@ void AliAnalysisTaskSEDs::CreateCutVarsAndEffSparses()
 
   Int_t nSparseAxes = knVarForSparse;
   if (!fApplyML)
-    nSparseAxes--;
+    nSparseAxes -= 3;
```

Comment:
Here you should differentiate between binary and multi-class classifications, because otherwise in case of binary classification you get 2 extra axes all filled with -1